<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsForm</class>
 <widget class="QWidget" name="SettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>1194</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="1" column="1">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>943</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="pushButtonFromPLC">
     <property name="text">
      <string>Прочитать</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="pushButtonToPLC">
     <property name="text">
      <string>Записать</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="4">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>925</width>
        <height>1123</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_10">
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>ПЛК</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_11">
           <item row="0" column="0">
            <widget class="QScrollArea" name="scrollArea_2">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="widgetResizable">
              <bool>true</bool>
             </property>
             <widget class="QWidget" name="scrollAreaWidgetContents_2">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>865</width>
                <height>1037</height>
               </rect>
              </property>
              <layout class="QGridLayout" name="gridLayout_7">
               <item row="0" column="0">
                <widget class="QGroupBox" name="groupBox_8">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>16777215</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="title">
                  <string/>
                 </property>
                 <layout class="QGridLayout" name="gridLayout_8">
                  <item row="0" column="0">
                   <widget class="QLabel" name="label">
                    <property name="text">
                     <string>Сетевой адрес:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="0" column="2">
                   <widget class="QSpinBox" name="spinBoxNetAddr">
                    <property name="maximum">
                     <number>255</number>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="0" colspan="2">
                   <widget class="QLabel" name="label_2">
                    <property name="text">
                     <string>Адрес при программировании:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="1" column="2">
                   <widget class="QSpinBox" name="spinBoxProgAddr">
                    <property name="maximum">
                     <number>255</number>
                    </property>
                   </widget>
                  </item>
                  <item row="2" column="0" colspan="3">
                   <widget class="QGroupBox" name="groupBox_2">
                    <property name="title">
                     <string/>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_4">
                     <item row="0" column="0">
                      <widget class="QLabel" name="label_3">
                       <property name="text">
                        <string>IP адрес:</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QLineEdit" name="lineEditIP">
                       <property name="inputMask">
                        <string>000.000.000.000;_</string>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="0">
                      <widget class="QLabel" name="label_4">
                       <property name="text">
                        <string>MAC адрес</string>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="1">
                      <widget class="QLineEdit" name="lineEditMAC">
                       <property name="inputMask">
                        <string>HH:HH:HH:HH:HH:HH;_</string>
                       </property>
                      </widget>
                     </item>
                     <item row="2" column="0">
                      <widget class="QLabel" name="label_12">
                       <property name="text">
                        <string>Шлюз:</string>
                       </property>
                      </widget>
                     </item>
                     <item row="2" column="1">
                      <widget class="QLineEdit" name="lineEditGATE">
                       <property name="inputMask">
                        <string>000.000.000.000;_</string>
                       </property>
                      </widget>
                     </item>
                     <item row="3" column="0">
                      <widget class="QLabel" name="label_14">
                       <property name="text">
                        <string>Маска:</string>
                       </property>
                      </widget>
                     </item>
                     <item row="3" column="1">
                      <widget class="QLineEdit" name="lineEditMASK">
                       <property name="inputMask">
                        <string>000.000.000.000;_</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item row="3" column="0" colspan="3">
                   <widget class="QGroupBox" name="groupBox_4">
                    <property name="title">
                     <string>канал PС</string>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_3">
                     <item row="0" column="0">
                      <widget class="QLabel" name="label_7">
                       <property name="text">
                        <string>Протокол:</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QComboBox" name="comboBoxPcProtocol">
                       <item>
                        <property name="text">
                         <string>RTU/BIN</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>ASCII</string>
                        </property>
                       </item>
                      </widget>
                     </item>
                     <item row="1" column="0">
                      <widget class="QLabel" name="label_8">
                       <property name="text">
                        <string>Скорость:</string>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="1">
                      <widget class="QComboBox" name="comboBoxPcSpeed">
                       <item>
                        <property name="text">
                         <string>4800</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>9600</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>19200</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>38400</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>57600</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>115200</string>
                        </property>
                       </item>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item row="4" column="0" colspan="3">
                   <widget class="QGroupBox" name="groupBox_3">
                    <property name="title">
                     <string>канал PROG</string>
                    </property>
                    <layout class="QGridLayout" name="gridLayout_2">
                     <item row="0" column="0">
                      <widget class="QLabel" name="label_5">
                       <property name="text">
                        <string>Протокол:</string>
                       </property>
                      </widget>
                     </item>
                     <item row="0" column="1">
                      <widget class="QComboBox" name="comboBoxProgProtocol">
                       <item>
                        <property name="text">
                         <string>RTU/BIN</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>ASCII</string>
                        </property>
                       </item>
                      </widget>
                     </item>
                     <item row="1" column="0">
                      <widget class="QLabel" name="label_6">
                       <property name="text">
                        <string>Скорость:</string>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="1">
                      <widget class="QComboBox" name="comboBoxProgSpeed">
                       <item>
                        <property name="text">
                         <string>4800</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>9600</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>19200</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>38400</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>57600</string>
                        </property>
                       </item>
                       <item>
                        <property name="text">
                         <string>115200</string>
                        </property>
                       </item>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item row="5" column="0">
                   <widget class="QLabel" name="label_11">
                    <property name="text">
                     <string>Расш. память польз (рег):</string>
                    </property>
                   </widget>
                  </item>
                  <item row="5" column="1" colspan="2">
                   <widget class="QSpinBox" name="spinBoxEMemSize">
                    <property name="minimum">
                     <number>10</number>
                    </property>
                    <property name="maximum">
                     <number>10000</number>
                    </property>
                   </widget>
                  </item>
                  <item row="6" column="0">
                   <widget class="QLabel" name="label_13">
                    <property name="text">
                     <string>описание проекта:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="7" column="0">
                   <widget class="QLineEdit" name="lineEditProjectDefenition"/>
                  </item>
                  <item row="8" column="0" colspan="3">
                   <widget class="QGroupBox" name="groupBox_6">
                    <property name="title">
                     <string/>
                    </property>
                    <layout class="QGridLayout" name="gridLayout">
                     <item row="0" column="0">
                      <widget class="QRadioButton" name="radioButtonNoEmulation">
                       <property name="text">
                        <string>Эмуляция отключена</string>
                       </property>
                       <property name="checked">
                        <bool>true</bool>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="0">
                      <widget class="QRadioButton" name="radioButtonInputEmulation">
                       <property name="text">
                        <string>Эмуляция входов</string>
                       </property>
                      </widget>
                     </item>
                     <item row="2" column="0">
                      <widget class="QRadioButton" name="radioButtonInputOutputEmulation">
                       <property name="text">
                        <string>Эмуляция входов/выходов</string>
                       </property>
                      </widget>
                     </item>
                     <item row="3" column="0">
                      <widget class="QCheckBox" name="checkBoxModbus">
                       <property name="text">
                        <string>Modbus Master эмуляция</string>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item row="9" column="0" colspan="3">
                   <widget class="QCheckBox" name="checkBoxDisplay">
                    <property name="text">
                     <string>Разрешить пульт на канале PU</string>
                    </property>
                   </widget>
                  </item>
                  <item row="10" column="0">
                   <widget class="QCheckBox" name="checkBoxSD">
                    <property name="text">
                     <string>Поддержка SD карты</string>
                    </property>
                   </widget>
                  </item>
                  <item row="11" column="0">
                   <widget class="QLabel" name="label_9">
                    <property name="text">
                     <string>Тип контроллера:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="11" column="1" colspan="2">
                   <widget class="QComboBox" name="comboBoxPLCType"/>
                  </item>
                  <item row="12" column="0">
                   <widget class="QLabel" name="label_10">
                    <property name="text">
                     <string>Порт программирования:</string>
                    </property>
                   </widget>
                  </item>
                  <item row="12" column="1" colspan="2">
                   <widget class="QComboBox" name="comboBoxPrPort"/>
                  </item>
                  <item row="13" column="0">
                   <widget class="QPushButton" name="pushButtonPortListUpdate">
                    <property name="text">
                     <string>Обновить список портов</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="0" column="1">
                <spacer name="horizontalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Заводские установки</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="0" column="0">
            <widget class="QGroupBox" name="groupBox_5">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="title">
              <string/>
             </property>
             <layout class="QGridLayout" name="gridLayout_5">
              <item row="0" column="0">
               <widget class="QRadioButton" name="radioButtonOneByte">
                <property name="text">
                 <string>1 байт</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QRadioButton" name="radioButtonTwoBytes">
                <property name="text">
                 <string>2 байта</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>143</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="2">
               <widget class="QRadioButton" name="radioButtonFourBytes">
                <property name="text">
                 <string>4 байта</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0" rowspan="3" colspan="4">
               <widget class="QTableWidget" name="tableWidget">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
